Data Structures/Graph.java

```diff
@@ -0,0 +1,60 @@
+public class Graph{
+    static class Node{
+        int data;
+        Node next;
+        Node(int data){
+            this.data = data;
+            this.next = null;
+        }
+    }
+    static class Edge{
+        int src;
+        int dest;
+        Edge(int src, int dest){
+            this.src = src;
+            this.dest = dest;
+        }
+    }
+
+        int V;
+        Node[] adjList;
+        Graph(int V){
+            this.V = V;
+            adjList = new Node[V];
+            for(int i = 0; i < V; i++){
+                adjList[i] = null;
+            }
+        }
+        public void addEdge(int src, int dest){
+            Node newNode = new Node(dest);
+            newNode.next = adjList[src];
+            adjList[src] = newNode;
+            newNode = new Node(src);
+            newNode.next = adjList[dest];
+            adjList[dest] = newNode;
+        }
+        public void printGraph(){
+            for(int i = 0; i < V; i++){
```

Comment:
give purpose-full variable name on iterations also

Reply:
And for temp named variable ,find its purpose what is scenario and then give another name

Reply:
Okay, sir, I will update the variable names to meaningful names.